jmespath/functions.py

```diff
@@ -2,6 +2,9 @@
 import json
 
 from jmespath import exceptions
+from jmespath.compat import get_methods
+from jmespath.compat import iteritems
+from jmespath.compat import map
 from jmespath.compat import string_type as STRING_TYPE
 from jmespath.compat import get_methods
```

Comment:
Looks like you wanted to sort the imports? But, the `get_methods` import is duplicated instead of moved.
```suggestion
from jmespath.compat import get_methods
from jmespath.compat import iteritems
from jmespath.compat import map
from jmespath.compat import string_type as STRING_TYPE
```

Reply:
Awesome thanks for your valued feedback.